src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
I can understand both arguments, and I flip-flop between them.

That said, I'm leaning a bit toward _maintaining_ the namespace, even for state classes.

The main reason is that Fabricator's JS toggles both _states_ and _utility classes_. So it feels weird to me to toggle `.f-u-hidden` one place but `.is-active` in another. And because `.f-u-hidden` needs to apply to many elements, it's not really realistic to convert it to a state (`.is-hidden`) because it will be more difficult and verbose to _attach_ it to namespaced classes to avoid collisions with toolkit CSS.

I also start thinking about explaining the distinction to new collaborators. It just seems simpler to me to say "if it's defined in Fabricator, it's namespaced."

@erikjung @lyzadanger Thoughts?